src/themes/elife/staticCustomProperties.css

```diff
@@ -49,4 +49,6 @@
   --URL-DOWNLOAD-ICON--NARROWER-VIEWPORTS: url('https://elifesciences.org/assets/patterns/img/icons/download.ecfa2d98.svg');
   --URL-DOWNLOAD-ICON--WIDER-VIEWPORTS: url('https://elifesciences.org/assets/patterns/img/icons/download-full.6691999e.svg');
   --URL-EMAIL-ICON: url('https://elifesciences.org/assets/patterns/img/icons/corresponding-author@1x.89247d49.png');
+  --URL-CC-ICON: url('https://elifesciences.org/assets/patterns/img/icons/cc.ec7b6e9c.svg');
```

Comment:
We should not be referencing assets in this way

Reply:
Will leave it in and address with: https://github.com/elifesciences/issues/issues/5911